src/ert/config/ert_script.py

```diff
@@ -113,6 +125,33 @@ def initializeAndRun(
                 arguments.append(None)
 
         try:
+            func_args = inspect.signature(self.run).parameters
+            for i, val in enumerate(func_args):
+                if val in fixtures:
```

Comment:
Is there a else statement to this if?

Reply:
No, not sure what that would be?

Reply:
So you specified a parameter to you run function that we will not be supplying, that seems like it will produce an error (at least if it is an positional argument)